docs: add authorization setting for dora webhook

mappjzc opened the pull request:
Add header for webhook dora curl
Add doc to show how to caculate authorization: Basic

mappjzc committed:
docs: add authorization setting for dora webhook

Add header for webhook dora curl
Add doc to show how to caculate authorization: Basic

Nddtfjiang <zhicheng.jiang@merico.dev>

mappjzc committed:
Merge branch 'main' into dora

mappjzc committed:
Update Dora.md

fix merge error

mappjzc committed:
Update DORA.md

Startrekzky commented on docs/UserManuals/DORA.md:
```diff
@@ -132,7 +132,10 @@ Using CircleCI as an example, we demonstrate how to actively push data to DevLak
               # Send the request to DevLake after deploy
               # The values start with a '$CIRCLE_' are CircleCI's built-in variables
-              curl https://sample-url.com/api/plugins/webhook/1/deployments -X 'POST' -d "{
+              curl https://sample-url.com/api/plugins/webhook/1/deployments -X 'POST' \
```
Just use 'username/password' is OK. For example,

curl https://sample-url.com/api/plugins/webhook/1/deployments -X 'POST' -u 'username:password' -d '{
    ...
    ...

Startrekzky commented on docs/UserManuals/DORA.md:
```diff
@@ -144,6 +147,14 @@ Using CircleCI as an example, we demonstrate how to actively push data to DevLak
+
+  ![](base64.jpg)
+
+  And after you caculated your `username:password` base64 coding.You have to use your base64 coding to replace `dXNlcm5hbWU6cGFzc3dvcmQ=`.
```
If you have set a [username/password](https://devlake.apache.org/docs/UserManuals/Authentication) for Config UI, you need to add the username and password to the curl to register a deployment:

```
curl https://sample-url.com/api/plugins/webhook/1/deployments -X 'POST' -u 'username:password' -d '{
    \"commit_sha\":\"$CIRCLE_SHA1\",
    \"repo_url\":\"$CIRCLE_REPOSITORY_URL\",
    \"start_time\":\"$start_time\"
  }'
```

mappjzc committed:
docs: header change

change header to be -u to set username and password.

Nddtfjiang <zhicheng.jiang@merico.dev>

Startrekzky reviewed (approved):
LGTM
